.eslintrc.json

```diff
@@ -62,12 +66,15 @@
     "no-restricted-exports": "off",
     "no-use-before-define": [2, "nofunc"],
     "no-confusing-arrow": "off",
-    "no-unused-vars": [2, { "vars": "local", "args": "none" }],
     "no-multi-str": 2,
     "no-promise-executor-return": "off",
     "no-irregular-whitespace": 2,
+    "@typescript-eslint/no-empty-function": "off",
     "comma-dangle": "off",
     "max-len": "off",
+    // TODO: Investigate if we can avoid the issue instead of
+    // deactivating the lint rule.
+    "import/no-duplicates": "off",
```

Comment:
Can you create an issue for this please?

AFAIK, this shouldn't be too much of a problem but to make the review and the processing of this PR easier, I'd rather have this in another task.

Link to the doc: https://eslint.org/docs/latest/rules/no-duplicate-imports

Reply:
Once the issue is created, you can replace this comment's content by the issue's link

Reply:
Done